apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx

```diff
@@ -308,6 +314,23 @@ const ExpandedCurriculumCatalogCard = ({
                   text={recommendedSimilarCurriculum.display_name}
                   onClick={handleClickRecommendedSimilarCurriculum}
                 />
+                <img
+                  id="stretchCurriculumImage"
+                  src={recommendedStretchCurriculum.image || defaultImageSrc}
+                  style={{height: '100%'}}
```

Comment:
nit: could this be moved to scss?

Reply:
Yep! I actually tested it and adding `height: 100%` to the `img` doesn't do anything here! So I'll just remove it